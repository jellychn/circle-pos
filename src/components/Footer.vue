<template>
  <div class="footer">
    <p class="copyright">Â© 2024 CirclePOS.com All Rights Reserved.</p>
  </div>
</template>

<style lang="scss" scoped>
.footer {
  width: 100%;
  padding: 20px;
  background-color: var(--vt-c-text-dark-2);

  .copyright {
    width: 100%;
    text-align: center;
    margin: auto;
  }
}
</style>