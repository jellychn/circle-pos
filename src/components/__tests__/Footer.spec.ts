import { shallowMount } from '@vue/test-utils'
import { describe, it, expect } from 'vitest'
import Footer from '@/components/Footer.vue'

describe('Footer.vue', () => {
  it('matches snapshot', () => {
    const wrapper = shallowMount(Footer)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
