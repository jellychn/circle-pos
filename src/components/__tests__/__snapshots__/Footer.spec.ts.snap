// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer.vue > matches snapshot 1`] = `
"<div data-v-4896eafd="" class="footer">
  <p data-v-4896eafd="" class="copyright">Â© 2024 CirclePOS.com All Rights Reserved.</p>
</div>"
`;
